integrationTests/vm/arwen/utils.go

```diff
@@ -84,11 +84,12 @@ type TestContext struct {
 	GasLimit    uint64
 	GasSchedule map[string]map[string]uint64
 
-	EpochNotifier     process.EpochNotifier
-	UnsignexTxHandler process.TransactionFeeHandler
-	EconomicsFee      process.FeeHandler
-	LastConsumedFee   uint64
-	ArwenChangeLocker common.Locker
+	EpochNotifier       process.EpochNotifier
```

Comment:
do we still need epoch notifier here?

Reply:
still needed for the remaining components that still use it